L1Trigger/Phase2L1ParticleFlow/interface/puppi/linpuppi_ref.h

```diff
@@ -211,6 +258,19 @@ namespace l1ct {
     std::vector<double> alphaSlope_, alphaZero_, alphaCrop_;
     std::vector<double> priorNe_, priorPh_;
     std::vector<pt_t> ptCut_;
+
+    // NNVtx Association:
+    bool useMLAssociation_;
+    double associationThreshold_;
+    std::string associationGraphPath_;
+    std::vector<double> associationNetworkZ0binning_, associationNetworkEtaBounds_, associationNetworkZ0ResBins_;
+    std::unique_ptr<NNVtxAssoc> nnVtxAssoc_;
+    tensorflow::GraphDef *associationGraph_;
+    tensorflow::Session *associationSesh_;
+#ifdef CMSSW_GIT_HASH
+    edm::ParameterSet nnVtxAssocPSet_;
```

Comment:
Can we create a constructor of `NNVtxAssoc` from PSet and avoid saving it as data-member?
The constructor could call the existing one without any code duplication and be protected by `CMSSW_GIT_HASH`

Reply:
Now removed storing the PSet as a member variable in `LinPuppiEmulator `.